src/main/java/com/bot/ShardingManager.java

```diff
@@ -50,6 +51,9 @@ public ShardingManager(int numShards, boolean supportScript) throws Exception {
                     // Battle Royale
                     new BattleRoyaleCommand());
 
+                    // General Commands
+                    new InviteCommand();
```

Comment:
nitpick: alphabetize with other commands

Reply:
Also remove the comments, code should be self documenting.